internal/locate/region_cache.go

```diff
@@ -786,10 +796,12 @@ func (l *KeyLocation) LocateBucket(key []byte) *Bucket {
 			l.EndKey,
 		}
 	}
-	return bucket
+	// unreachable
+	logutil.Eventf(context.Background(), "Unreachable place: Region: [%v, %v), Key: %v", l.StartKey, l.EndKey, key)
```

Comment:
Use `logutil.Logger()` is enough here. Try to use `hex.EncodeToString` to wrap the keys.